validator/keymanager/remote-web3signer/testing/good_keyfile.txt

```diff
@@ -0,0 +1,3 @@
+0x8000a9a6d3f5e22d783eefaadbcf0298146adb5d95b04db910a0d4e16976b30229d0b1e7b9cda6c7e0bfa11f72efe055
+
+0x800057e262bfe42413c2cfce948ff77f11efeea19721f590c8b5b2f32fecb0e164cafba987c80465878408d05b97c9be
```

Comment:
Add a key without the `0x` prefix?

Reply:
can update I did test it locally with it and I think some unit test has it too